File: JAVA/uberPool.java
public class uberPool extends car {
    public String brand;
    public String model;

    public uberPool(String license, driver driver, int passenger, String brand, String model) {
        super(license, driver, passenger);
        this.brand = brand;
        this.model = model;
    }

    @Override
    public String toString() {
        return (super.toString() + ", Brand: " + brand + ", Model: " + model);
    }
}

File: PHP/uberPool.php
<?php
require_once('car.php');
class uberPool extends car {
    public $brand;
    public $model;

    public function __contructor($license, $driver, $passenger, $brand, $model) {
        parent::__contructor($license, $driver, $passenger);
        $this->brand = $brand;
        $this->model = $model;
    }

    public function printDataCar() {
        return parent::printDataCar() + ", Brand: $this->brand, Model: $this->model";
    }
}
?>